### ISSUE TITLE
Operation not supported. MFE_CR_ERROR

### ISSUE THREAD

**Issue by rajil** [Mellanox/mstflint#737]
Hello,

I have a dual port card which shows up as

```
# /sbin/lspci -d 15b3:
01:00.0 Ethernet controller: Mellanox Technologies MT26448 [ConnectX EN 10GigE, PCIe 2.0 5GT/s] (rev b0)
```

The card has a part number of 81Y 1541 which shows up as IBM CONNECTX-2 card.

I wanted to check the firmware version, but it doesnt work.

```
# insmod ../kernel/mstflint_access.ko
# ./mstflint -d 01:00.0 q
-E- Cannot open Device: 01:00.0. Operation not supported. MFE_CR_ERROR
```

```
# ethtool -i ens1  
driver: mlx4_en
version: 4.0-0
firmware-version: 2.7.700
expansion-rom-version: 
bus-info: 0000:01:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: no
supports-register-dump: no
supports-priv-flags: yes



# ethtool  ens1
Settings for ens1:
        Supported ports: [ TP ]
        Supported link modes:   10000baseT/Full
        Supported pause frame use: No
        Supports auto-negotiation: No
        Supported FEC modes: Not reported
        Advertised link modes:  10000baseT/Full
        Advertised pause frame use: No
        Advertised auto-negotiation: No
        Advertised FEC modes: Not reported
        Speed: 10000Mb/s
        Duplex: Full
        Auto-negotiation: off
        Port: Twisted Pair
        PHYAD: 0
        Transceiver: internal
        MDI-X: Unknown
        Supports Wake-on: d
        Wake-on: d
        Current message level: 0x00000014 (20)
                               link ifdown
        Link detected: yes
```

**Comment by ahmadawwad** [comment]
Hi, 
connectX2 is out of the supported device [very old device]
